contracts-periphery/Makefile

```diff
@@ -0,0 +1,8 @@
+-include .env
+
+forkBlock = 14800081
+
+build :; FOUNDRY_PROFILE=hook forge build
+install :; forge install
+Test :; FOUNDRY_PROFILE=hook forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-contract WithdrawData -vv
+Testv :; Test :; FOUNDRY_PROFILE=hook forge test -f ${ETH_RPC_URL} --fork-block-number ${forkBlock} --match-contract WithdrawData -vvvv
```

Comment:
I don't think this is the makefile setup we want. With this I'd run `make Test` to run hook tests and presumably `forge test` to run all tests. But since these tests require forking, `forge test` will fail. We should add `test-{description}` lines for each type we want

Reply:
Agree! Also not sure I understand the capital T pattern here?